pkg/cmdutil/config.go

```diff
@@ -4,7 +4,7 @@
 package cmdutil
 
 import (
-	"github.com/Masterminds/semver/v3"
+	image2 "github.com/oracle-cne/ocne/pkg/image"
```

Comment:
this looks like a duplicate of line 12

Reply:
Thanks! I have edited changed the image parameter to imageForContainer to resolve this issue.